### DISCUSSION TITLE
Duplicate Pub feature?

### DISCUSSION THREAD

**Discussion by jeffpooley** [knowledgefutures/pubpub#1412] [Ideas]
I have a question for the community: Is there any interest in a Pub duplication feature? The use case I have in mind for our small book/journal publisher community is to have a private Collection with what are, in effect, templates for article/chapter types. The ability to duplicate such Pubs would make such templating far more useful.

**Comment by gabestein** [comment]
Thanks Jeff! We're interested in the idea of both duplicating and templates, though have typically thought of them as two slightly different things. Typically, we've been thinking of tying Pub templates to a submission button, and duplicating as a dashboard feature.

Are you mainly looking at it from the templating perspective, or are there other uses for duplication? And also, is it part of submissions for you, or is it more about internal workflow where you just want to be able to quickly duplicate things you (or internal folks) produce a lot?

**Reply by jeffpooley** [reply]
Thanks Gabe. Absolutely, on duplicating and templates as two different things. The kind of template Collection idea, with template Pubs to be duplicated, is a hacky workaround. Far better would be the ability to create templates as part of a future submission process—as long as signed-in Community admins, etc., have the ability to use the submission button too. Duplication, on the dashboard, would be welcome for its own sake, for a range of uses that have nothing to do with templates.

**Comment by kyledenlinger** [comment]
[The roadmap seems to suggest](https://github.com/orgs/pubpub/projects/9#card-52162132) that "admin-authored submission instructions and Pub body templates" are now possible. I don't see other documentation of these features -- are they live?

I agree with @jeffpooley -- a template should be what users get when they go to make a submission. Duplication (of individual pubs or entire communities) would be immensely helpful from an OER standpoint, especially since all Pubs bear a CC license. Make it easy for folks to take advantage of those licenses.

Duplication could automatically relate the original and the duplicate through a Connection. "This Pub is a _derivative_ of the original at ######", thus fulfilling attribution requirements. A feature like that would be absolutely killer.

I also think it should be easier to create responses to Pubs and connect them across communities. Medium's Responses feature is generally what I have in mind -- somewhere between a comment and a standalone publication.

**Reply by gabestein** [reply]
Thanks for the feedback!

Submissions with instructions are available, yes. You can turn it on from any collection vis the "Submissions" tab. Templates didn't make it into the first version.

Love the ideas about making it easier to create connections. That's certainly on our radar in general (the first step I'd like to see personally is turning any link into a connection without needing to visit settings); and the idea about how to create a duplicate (license-permitting) with a connection makes a lot of sense. We'll discuss in our next sprint meeting!
